.circleci/test.Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM reactioncommerce/base:latest as builder
```

Comment:
Related to a previous comment around testing... I don't think it's the responsibility of this project to run a `FROM` test. Everything below is downstream usage of this image. Such testing could (and should) be done in the reaction project, though!

Reply:
As mentioned above, this testing ensures that this base image works with the current stable release of Reaction _before_ it gets released in a potentially broken state.  I guess I don't see the downside of running tests of real world usage here.